src/System.Net.Sockets/tests/FunctionalTests/DualModeSocketTest.cs

```diff
@@ -620,8 +620,7 @@ public void DualModeConnectAsync_Static_DnsEndPointToHost_Helper(IPAddress liste
     [Trait("IPv6", "true")]
     public class DualModeBind : DualModeBase
     {
-        [Fact] // Base case
-        // "The system detected an invalid pointer address in attempting to use a pointer argument in a call"
+        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsSubsystemForLinux))] // https://github.com/Microsoft/BashOnWindows/issues/982
```

Comment:
I was trying to avoid viewing this as a separate platform, since it's supposed to behave like "real" Ubuntu.  If a test applies (or not) to Linux, it should apply (or not) in WSL as well.  These conditionals are the "rare" exceptions.